hil/ext/switches/ovs.py

```diff
@@ -0,0 +1,271 @@
+
+# Copyright 2013-2017 Massachusetts Open Cloud Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS
+# IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied.  See the License for the specific language
+# governing permissions and limitations under the License.
+"""A switch driver for the juniper QFX series.
+
+But in theory can work for any Juniper switch that supports NETCONF.
+Driver uses the python library PyEZ made available by Juniper.
+By default it sets NETCONF session over ssh, i.e. make RPC calls over ssh.
+For this to work, make sure that the Juniper switch is configured
+appropriately to recieve NETCONF connections. Its default port is 830.
+"""
+
+import re
+import logging
+import schema
+import sys
+import os
+import ast
+from commands import getoutput, getstatusoutput
+
+from hil.model import db, Switch, BigIntegerType
+from hil.migrations import paths
+from os.path import dirname, join
+logger = logging.getLogger(__name__)
+paths[__name__] = join(dirname(__file__), 'migrations', 'ovs')
```

Comment:
Don't worry about providing migrations. I think I mentioned earlier that re-running 'hil-admin db create' adds additional tables.

Reply:
I suppose that will need this line for it to function properly.
Since all drivers have this functionality I added it for consistency.

Reply:
from your [comment](https://github.com/CCI-MOC/hil/pull/850/commits/6bdde7d4122e2d759078114903a79c08568f66ed#r135841501) on the juniper PR. 

"I tested it, if a new driver is introduced to a running HIL installation it does not require a migration script. To add the new table to the database, stop the HIL servers and simple run hil-admin db create again. It will create any new tables based on the changes to the hil."

> I suppose that will need this line for it to function properly.
Since all drivers have this functionality I added it for consistency.

The paths variable is only needed for migrations (it is imported from hil.migrations in line 33). So there's no point having it here at this time.